views/send.erb

```diff
@@ -0,0 +1,61 @@
+<%require 'digest/md5'%>
+Logged in as <%= @current_username %>
+<br>
+<form action="/logout" method="post">
+  <input type="submit" value="Logout">
+</form>
+<br>
+<br>
+<center>
+  <%if !!@peeps%>
+    <% @peeps.each do |peep| %>
+    <br>
+    <br>
+    <% 
+    user_colour = (Digest::MD5.hexdigest peep.username)[0..5]
```

Comment:
This is a lot of logic to live in the views. You could have this code live in the User model, and then have the colour as a method of the user maybe.